src/Form/FormElementCollection.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\JsonSchema\FormBuilder\Form;
+
+use ArrayObject;
+use Illuminate\Support\Arr;
+
+class FormElementCollection extends ArrayObject
```

Comment:
```suggestion
/**
 * @extends ArrayObject<array-key, FormElement>
 */
class FormElementCollection extends ArrayObject
```
A subject for thought: the FormElementCollection and Form classes look very similar. My intuition tells me that there could be something about them here. Not sure yet. How about do you feel about it?